src/Fixer/Alias/NoMixedEchoPrintFixer.php

```diff
@@ -24,14 +22,9 @@
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  * @author SpacePossum
  */
-final class NoMixedEchoPrintFixer extends AbstractFixer implements ConfigurableFixerInterface
+final class NoMixedEchoPrintFixer extends AbstractFixer
```

Comment:
This is because the interface is implemented by `AbstractFixer` now, but I'm going to revert that change.